src/routes/SEO.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+	import { seo } from '$lib/stores/SeoStore';
+
+	export let title = 'listd';
```

Comment:
Change to
```ts
export const let title = $seo.title
export const let description = $seo.description
```

It already has sane defaults in the store file.

Reply:
Updated